examples/OECC/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenEdge Command Center #
+
+### Requirements 
+* Ubuntu Linux 22.04
+* OpenEdge media:
+    * PROGRESS_OE_12.8.3_LNX_64.tar.gz
+    * PROGRESS_OECC_SERVER_1.3.0_LNX_64.tar.gz
+    * PROGRESS_OECC_AGENT_1.3.0_LNX_64.bin
+* Reponse file for OpenEdge installation:
+    * response_12.8.3.ini
+    * Products:
+        * OE Adv. Ent. RDBMS
+        * 4GL Development System
+        * Progress ProdAS for OE
+
+**Notes:**
+* You can use the OpenEdge-Samples/examples/OECC/files/response_12.8.3.ini file as a template and specify the control codes in the file.
+
+### Components 
+* Ubuntu Virtual Machine
+    * OpenEdge Database
+    * PAS for OpenEdge
+    * OpenEdge Command Center agent
+    * Docker
+        * OpenEdge Command Center server
+        * MongoDB
+        * Prometheus
+        * Grafana
+
+![Components](./diagram.png)
+
+### Steps
+1. Create /files directory used for the installation files.
```

Comment:
What are "files" here? Are they configuration files? Would a "config" folder be more clear?

Reply:
The /files directory is used for the OpenEdge media, the response.ini file and configuration files tailored during the setup.